src/main.rs

```diff
@@ -391,17 +391,24 @@ fn process_options(args: &Args) {
     for option in &args.option {
         // Find the matching option in OPTIONS
         if let Some(option_item) = OPTIONS.iter().find(|item| item.name() == *option) {
-            // Check if the chip is supported. If the chip list is empty, all chips are supported
+            // Check if the chip is supported. If the chip list is empty,
+            // all chips are supported:
             if !option_item.chips().iter().any(|chip| chip == &args.chip)
                 && !option_item.chips().is_empty()
             {
                 eprintln!(
-                    "Error: Option {:?} is not supported for chip {:?}",
+                    "Error: Option '{}' is not supported for chip {}",
                     option, args.chip
                 );
+                process::exit(-1);
             }
         }
     }
+
+    if args.option.contains(&String::from("ble")) && args.option.contains(&String::from("wifi")) {
```

Comment:
That's fine but I was hoping we could detect such a situation more generally by checking the `disables` field in `GeneratorOption`

But given this is not a general-purpose library to create project generators it's probably unnecessary to deal with that

Reply:
I did consider a more thorough solution, but for the time being this at least solve the linked issue. Can revisit it at a later date.